samples/sample_hevc_fei/src/sample_hevc_fei.cpp

```diff
@@ -701,6 +712,11 @@ void AdjustOptions(sInputParams& params)
         msdk_printf(MSDK_STRING("WARNING: Encoded order is enabled by force in repackctrl.\n"));
         params.bEncodedOrder = true;
     }
+
+    if (!params.bExtBRC && params.TargetKbps) {
+        msdk_printf(MSDK_STRING("WARNING: Target bitrate is ignored as external BRC is disabled\n"));
+        params.TargetKbps = 0;
+    }
```

Comment:
What if I set ExtBRC and QP?

Reply:
Not sure. Is it OK to return with error, if they are set together?

Reply:
Yep, because you can't read users mind